viewer/src/main/webapp/WEB-INF/jsp/app.jsp

```diff
@@ -98,9 +98,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                     <script type="text/javascript" src="${contextPath}/viewer-html/common/openlayers/OpenLayers.js"></script>
                 </c:otherwise>
             </c:choose>
+            <script type="text/javascript" src="${contextPath}/viewer-html/common/overrides.js"></script>
         </c:if>
-        <script type="text/javascript" src="${contextPath}/viewer-html/common/overrides.js"></script>
         <script type="text/javascript" src="${contextPath}/resources/js/FileSaver.js"></script>
+            
+        <c:if test="${actionBean.viewerType == 'openlayers5'}">
+               <link href="${contextPath}/viewer-html/common/ol/ol.css" rel="stylesheet">  
+               <link href="${contextPath}/viewer-html/common/resources/css/openlayers.css" rel="stylesheet">
+               <script src="https://cdnjs.cloudflare.com/ajax/libs/proj4js/2.3.6/proj4.js" type="text/javascript"></script>
```

Comment:
I think its better to include this JS in our own source instead of depending on an external service